client/apps/articles_list/client.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { data as sd } from 'sharify'
+import { Provider } from 'react-redux'
+import { createReduxStore } from 'client/lib/createReduxStore'
+import { reducers, initialState } from 'client/reducers'
+import { init as initWebsocket } from 'client/apps/websocket/client'
+
+import ArticlesList from './components/articles_list'
+import Channel from 'client/models/channel.coffee'
+
+const store = createReduxStore(reducers, initialState)
+initWebsocket(store)
+
+export const init = () => (
+  render(
+    <Provider store={store}>
+      <ArticlesList
+        articles={sd.ARTICLES}
+        published={sd.HAS_PUBLISHED}
+        channel={new Channel(sd.CURRENT_CHANNEL)}
```

Comment:
The channel is also now in the store here - https://github.com/artsy/positron/blob/master/client/reducers/appReducer.js#L4

Reply:
Oh I must've missed it during the merge, cool I'll remove that